app/controllers/report_controller/reports/editor.rb

```diff
@@ -131,11 +131,24 @@ def miq_report_add_edit
     replace_right_cell
   end
 
+  # Get string with unavailable fields while adding/editing report
+  def unavailable_fields_for_model(model)
+    case model
+    when 'ChargebackVm'
+      _('CPU Cores Allocated Metric, CPU Cores Used Metric, Fixed Compute Metric, Fixed Storage Metric')
+    when 'ChargebackContainerImage'
+      _('CPU Allocated Metric, CPU Used Metric, Disk I/O Used Metric, Fixed Compute Metric, Fixed Storage Metric, Storage Allocated Metric, Storage Used Metric')
```

Comment:
please remove `CPU Cores Used Metric`,  `Fixed Compute Metric`, we have it for ChargebackContainerImage

Reply:
I don't see `CPU Cores Used Metric` for 'ChargebackContainerImage'. Did you mean `CPU Used Metric`? Or did you mean to remove `CPU Cores Used Metric` for 'ChargebackVm'? Thanks! :)